### ISSUE TITLE
3일 평가판 관련 문의

### ISSUE THREAD

**Issue by Sienna-lee** [edh-git/EL_Display_Hub#1256]
- 문의 날짜 : 2022년 7월 8일 오전 5시 48분
- 메일 주소 : jennyjennyjenny27@gmail.com

Title: Just doesn\'t work - will I be charged after 3 day trial?

Questions: Hi there
I\'m not really sure how to describe the problem. I\'ve downloaded EL display hub on my PC (Windows 10 Home) and the EasyCanvas app on my iPad (iOS 10.3.3), connected the iPad to the laptop with a USB cable, but nothing happens. I just have a black screen on the iPad, apart from the shortcuts bar. I\'ve tried restarting etc but I just can\'t get it to work. I don\'t really want to spend ages trying to get it to work - I just want to check that I won\'t be charged for it. I clicked on \"free 3 day trial\" but it wasn\'t clear whether I was then going to be charged automatically after the trial period ended.
To summarise: I don\'t want to use the app so could you please confirm that I won\'t be charged for it after the 3-day trial?
Many thanks in advance
Jennifer

**Comment by Sienna-lee** [comment]
해당 사용자에게 아래와 같이 회신하였습니다.

----

Hello, customer. We are an Easy & Light support team.

I'm sending this email for your inquiry.



Did you buy EasyCanvas(in-app)?
After the 3days trial, payment is not automatically made, so just leave it.


If you have purchased EasyCanvas subscription, please refer to the link below.

https://support.apple.com/en-us/HT202039


If payment has been made, you can request a refund from Apple.

https://support.apple.com/en-us/HT204084



Thank you.

Easy&Light
